### PR TITLE
Tweaks for single release pipeline

### PR DESCRIPTION
**Pull request by adrinr**
## Description
A few changes needed for the single release pipeline and some cleanups:
1. Adding script to set versions not dependant on lerna.json
2. Increasing timeouts on yarn install in dockers
3. Cleanup test pipelines
4. Cleanup npm develop scripts

### CODE REVIEW

File: .github/workflows/release-singleimage-test.yml
Comment (mike12345567) [inline .github/workflows/release-singleimage-test.yml:1]: Out of interest, why was this removed?

**Review by mike12345567 — APPROVED** [review]
LGTM - just one question!

File: .github/workflows/release-singleimage-test.yml
Comment (adrinr) [reply .github/workflows/release-singleimage-test.yml:1]: This is moved now to budibase-deploys, it was added here just to see that everything works fine
